lib/data/wrapper.js

```diff
@@ -96,31 +96,36 @@ const data = {
         // If objectGetInfo.key exists the md-model-version is 2 or greater.
         // Otherwise, the objectGetInfo is just the key string.
         const key = objectGetInfo.key ? objectGetInfo.key : objectGetInfo;
-        log.debug('sending delete to datastore', { implName, key,
+        log.debug('sending delete to dataStore', { implName, key,
```

Comment:
Just trying to be consistent; I saw another error trace used dataStore (former line 385 of services.js) and opted to use the camelcase. If lowercase is preferred because it's not an entity except in concept, I can revert back to datastore :)